dandi/girder.py

```diff
@@ -150,8 +158,11 @@ def dandi_authenticate(self):
                     "in Girder) for {}: ".format(client_name)
                 )
                 keyring.set_password(app_id, "key", api_key)
+                lgr.debug("Stored key in keyring")
+
             try:
                 self.authenticate(apiKey=api_key)
+                lgr.debug("Successfully authenticated using the key")
                 break
             except Exception as exc:
                 sys.stderr.write("Failed to authenticate: {}".format(exc))
```

Comment:
hm, I have not realized that we are not testing this since we always just rely on the env var... oh well -- girder will be gone soon, so we have better things to worry about and let's pretend that this code could not be the reason for #238 ;-)